packages/wrapped-keys/src/lib/constants.ts

```diff
@@ -0,0 +1,31 @@
+const CHAIN_ETHEREUM = 'ethereum';
+const LIT_PREFIX = 'lit_';
+export const NETWORK_EVM = 'evm';
+export const NETWORK_SOLANA = 'solana';
+export type Network = typeof NETWORK_EVM | typeof NETWORK_SOLANA;
+
+// Update the endpoint to Wrapped Key project endpoint
```

Comment:
can we at least add a link to the PR in the comment for now?

Reply:
There's no PR yet

Reply:
ref: https://github.com/LIT-Protocol/wrapped-keys-lambdas/pull/3